### ISSUE TITLE
bitru: Got redirected to another domain

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#9199]
The site is working fine in the browser but not in Jackett. I got 0 results.
```
{0} Jackett.Common.IndexerException: Exception (bitru): Got redirected to another domain. Try changing the indexer URL to https://yandex.ru/.
 ---> System.Exception: Parse error
 ---> Jackett.Common.ExceptionWithConfigData: Got redirected to another domain. Try changing the indexer URL to https://yandex.ru/.
   at Jackett.Common.Indexers.CardigannIndexer.CheckIfLoginIsNeeded(WebClientStringResult Result, IHtmlDocument document) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 841
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1355
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 831
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1684
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 347
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 347
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 808
   at Jackett.Server.Controllers.ResultsController.Results(ApiSearch requestt) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/ResultsController.cs:line 224
```

**Comment by garfield69** [comment]
this should have been fixed with #9172 
are you on the latest jackett?

**Comment by ngosang** [comment]
yes, i'm

**Comment by garfield69** [comment]
must have changed again. ill take a look.

**Comment by garfield69** [comment]
working for me
![image](https://user-images.githubusercontent.com/6933240/88106297-5c0ef200-cbf9-11ea-8697-f714f6814d6c.png)

**Comment by ilike2burnthing** [comment]
I had to delete bitru, restart Jackett, and add bitru again for it to work. Sorry, I thought it was just limited to me.

**Comment by ngosang** [comment]
> I had to delete bitru, restart Jackett, and add bitru again for it to work. Sorry, I thought it was just limited to me.

Working after doing that. Any idea how to fix for all users without manual intervention?

**Comment by garfield69** [comment]
no idea. presumably there is a cookie key/value that needs removing? way to much effort to code a parser to clean the bitru.json for this one event IMO.

**Comment by ngosang** [comment]
ok

**Comment by GAS85** [comment]
As a quicker solution, go to edit, added to Site Link some stuff, e.g. `http://bitru.org/kkk/` and save it.
Then edit again and put correct Site Link: `http://bitru.org/` and then it simply works without need to delete it and create again.

**Comment by Kedryn** [comment]
I have the same problem. Tried removing and re-adding, and tried changing url, but i keep getting same error.
i'm on latest nightly

**Comment by ilike2burnthing** [comment]
Did you restart Jackett after removing and before re-adding?

**Comment by Kedryn** [comment]
Yes. Still error.

**Comment by ilike2burnthing** [comment]
To be clear, what is the version number at the bottom of Jackett's page?

**Comment by Kedryn** [comment]
Jackett Version 0.16.734.0

**Comment by ilike2burnthing** [comment]
Latest Jackett is 0.16.962.0, fix was added in 0.16.857
